src/mono/mono/mini/helpers.c

```diff
@@ -76,6 +76,7 @@ static const gint16 opidx [] = {
 #endif
 //#define ARCH_PREFIX "powerpc64-linux-gnu-"
 
+#ifndef DISABLE_LOGGING
```

Comment:
@sayeedkhannabil thanks for looking into this!

This looks good. However if we surround the function definition with `#ifndef DISABLE_LOGGING` then we do not need the check for the same preprocessor directive inside of its body, and the `#else` is not needed anymore. 

So I believe something like:
```c
#ifndef DISABLE_LOGGING
const char*
mono_inst_name (int op) {
	if (op >= OP_LOAD && op <= OP_LAST)
		return (const char*)&opstr + opidx [op - OP_LOAD];
	if (op < OP_LOAD)
		return mono_opcode_name (op);
	g_error ("unknown opcode name for %d", op);
	return NULL;
}
#endif
```
would be cleaner.

Reply:
@ivanpovazan thanks for the suggestion. I have updated it.